app/models/miq_server.rb

```diff
@@ -491,15 +491,15 @@ def self.managed_resources
     {
       :vms                     => Vm.active.count,
       :hosts                   => Host.active.count,
-      :aggregate_physical_cpus => MiqRegion.my_region.aggregate_physical_cpus(Host.active),
+      :aggregate_physical_cpus => Host.active.first.aggregate_physical_cpus,
```

Comment:
I don't understand this change here, specifically the change of (what seems to be) removing the scoping of `MiqRegion.my_region` here.  Can you explain?